### PR TITLE
Updated the Date details

### PR DESCRIPTION
**Pull request by samagragupta**
date

### CODE REVIEW

File: src/components/Panel/index.js [hunk 1]
```diff
@@ -42,7 +42,8 @@ class Panel extends React.Component {
         <h3 className="uk-card-title white">{this.props.event.name}</h3>
         <p className={'uk-text-left panel-team-size primary-color'}>TEAM SIZE: <b>{this.props.event['team-size']}</b>
         </p>
-        <p className={'uk-text-left panel-team-size primary-color'}>{dateString}</p>
+        {/* <p className={'uk-text-left panel-team-size primary-color'}>{dateString}</p> */}
+        <p className={'uk-text-left panel-team-size primary-color'}>Date : Will be updated soon!</p>
```
Comment (aashutoshrathi) [inline src/components/Panel/index.js]: Make it `Details will be updated soon!`

Comment (samagragupta) [reply src/components/Panel/index.js]: Done 😄

File: src/components/Panel/index.js [hunk 2]
```diff
@@ -42,7 +42,8 @@ class Panel extends React.Component {
         <h3 className="uk-card-title white">{this.props.event.name}</h3>
         <p className={'uk-text-left panel-team-size primary-color'}>TEAM SIZE: <b>{this.props.event['team-size']}</b>
```
Comment (aashutoshrathi) [inline src/components/Panel/index.js]: Comment this line too.

Comment (samagragupta) [reply src/components/Panel/index.js]: Commented

**Pull request merged**